share/ruby-build/truffleruby-19.3.0.2

```diff
@@ -0,0 +1,13 @@
+case $(uname -s) in
+Linux)
+  install_package "truffleruby-19.3.0.2" "https://github.com/oracle/truffleruby/releases/download/vm-19.3.0.2/truffleruby-19.3.0.2-linux-amd64.tar.gz" truffleruby
```

Comment:
This is missing the checksums, I added them in 4082f9eff2f4cdd84ec1b512f0b2046458266e42

Reply:
I wasn't sure what those were. Thanks for fixing this @eregon

Reply:
For future reference, what type of checksums are those and how did You generate proper hash?

Reply:
I use `script/update-truffleruby` documented in https://github.com/oracle/truffleruby/blob/master/doc/contributor/updating-ruby-installers.md
Then the checksums are automatically computed.

In general I prefer to push updates to Ruby installers myself because it's more efficient and consistent (I have push rights on all 3 Ruby installers).

Reply:
My bad! Interesting thing is that I just checked https://github.com/oracle/truffleruby/releases and commit for 19.3.0.2 is exactly the same as for 19.3.0 . But checksums differ between these versions.

Reply:
Yeah it was rebuilt anyway, it could matter e.g., if there was a Graal compiler or Truffle fix in there.